tests/test_fixtures/rule_based_profiler/alice_user_workflow_verbose_profiler_config.yml

```diff
@@ -36,27 +43,25 @@ rules:
       - expectation_type: expect_column_values_to_be_between
         class_name: DefaultExpectationConfigurationBuilder
         module_name: great_expectations.rule_based_profiler.expectation_configuration_builder
-        min_value: $parameter.my_min_user_id.value[-1]
-        max_value: $variables.max_user_id
+        min_value: $variables.very_small_user_id
+        max_value: $variables.very_large_user_id
         column: $domain.domain_kwargs.column
       - expectation_type: expect_column_values_to_not_be_null
         class_name: DefaultExpectationConfigurationBuilder
         module_name: great_expectations.rule_based_profiler.expectation_configuration_builder
         column: $domain.domain_kwargs.column
-      - expectation_type: expect_column_values_to_be_between
+      - expectation_type: expect_column_values_to_be_less_than
         class_name: DefaultExpectationConfigurationBuilder
         module_name: great_expectations.rule_based_profiler.expectation_configuration_builder
-        min_value: $parameter.my_min_user_id.value[-1]
-        max_value: $variables.max_user_id
+        value: $parameter.my_max_user_id.value[-1]
         column: $domain.domain_kwargs.column
-        condition: $parameter.my_min_user_id.value[-1] > 0
-      - expectation_type: expect_column_values_to_be_between
+        condition: $parameter.my_max_user_id.value[-1] < $variables.very_large_user_id
+      - expectation_type: expect_column_values_to_be_greater_than
         class_name: DefaultExpectationConfigurationBuilder
         module_name: great_expectations.rule_based_profiler.expectation_configuration_builder
-        min_value: $parameter.my_min_user_id.value[-1]
-        max_value: $variables.max_user_id
+        value: $parameter.my_min_user_id.value[-1]
         column: $domain.domain_kwargs.column
-        condition: $parameter.my_min_user_id.value[-1] > 0 & $variables.max_user_id > 0
+        condition: $parameter.my_min_user_id.value[-1] > 0 & $parameter.my_min_user_id.value[-1] > $variables.very_small_user_id
```

Comment:
We are no longer re-using parameters for a given expectation type and we are editing more expectation types.